src/lib/validations/is-multisite-domain-mapped.js

```diff
@@ -76,8 +76,11 @@ export async function isMultisitePrimaryDomainMapped(
 	envId: number,
 	primaryDomain: string,
 ): Promise<boolean> {
-	const track = trackEventWithEnv.bind( null, appId, envId );
+	if ( ! primaryDomain ) {
+		return true;
```

Comment:
So if there is no primary domain, we assume that the primary domain is mapped?

Reply:
If the import file doesn't update the `wp_site` table, then there's nothing to validate at this stage of the import process. In this scenario we just want the validation to pass.

Reply:
Would it be clearer to have a separate `doesPrimaryDomainExist()` function and then use that here, or in the final check in `site-type.js`

Reply:
It would! Done in [5cb8c33](https://github.com/Automattic/vip/pull/1078/commits/5cb8c33e1efdda5e8e85077fc6cc448090c8ef65)